rakelib/form526.rake

```diff
@@ -174,14 +174,21 @@ namespace :form526 do
     print_errors(errors)
   end
 
-  desc 'Get one or more submission details given an array of ids'
+  desc 'Get one or more submission details given an array of ids (either submission_ids or job_ids)'
   task submission: :environment do |_, args|
     raise 'No submission ids provided' unless args.extras.count.positive?
 
+    def integer?(obj)
+      obj.to_s == obj.to_i.to_s
+    end
     Rails.application.eager_load!
 
     args.extras.each do |id|
-      submission = Form526Submission.find(id)
+      submission = if integer?(id)
+                     Form526Submission.find(id)
+                   else
+                     Form526JobStatus.where(job_id: id).first.form526_submission
+                   end
```

Comment:
Rather than check each array value to see if it's a `submission_ids` or a `job_id` - could we instead have 2 arrays passed in as separate arguments?